src/CSS/AddItemForm.css

```diff
@@ -0,0 +1,30 @@
+/*Name*/
```

Comment:
A common practice with React apps is to keep component stylesheets adjacent to the components that consume them. What would you think of moving this file next to `AddItemForm.js`?